src/core/internationalization.js

```diff
@@ -1,5 +1,7 @@
 import i18next from 'i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
+import { version } from '../../package.json';
+
```

Comment:
Browserify used by the 1.x branch does not treeshake JSON which means the whole `package.json` file will be included in the build with this. We have the version number exported by [`core/constants.js`](https://github.com/processing/p5.js/blob/main/src/core/constants.js#L14-L15) so we can import that and use that value directly instead.

Reply:
Thanks! Updated.